partcad/src/partcad/part_config_manufacturing.py

```
@@ -0,0 +1,35 @@
+class PartConfigManufacturing:
+    method: int | None
+
+    def __init__(self, final_config):
+        manufacturing_config = final_config.get("manufacturing", {})
+        method_string = manufacturing_config.get("method", None)
+        if method_string == "additive":
+            self.method = METHOD_ADDITIVE
+        else:
+            self.method = METHOD_NONE
+
+    def _method_string(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_method_string`

Add return type annotation: `str`

(ANN202)